code/modules/cargo/packs.dm

```diff
@@ -38,6 +38,7 @@
 			new item(C)
 
 // If you add something to this list, please group it by type and sort it alphabetically instead of just jamming it in like an animal
+//A, B, C, D, E, F, G, H, I, J, K, L, M, N, O, P, Q, R, S, T, U, V, W, X, Y, Z <-- For quick and easy references
```

Comment:
Congrats, you understand the alphabet I guess??

Reply:
It helped me a bit.